lib/core/src/lib/app-config/app.config.schema.json

```diff
@@ -940,6 +940,10 @@
       "description": "BPM host",
       "type": "string"
     },
+    "devMode": {
```

Comment:
you are introducing an undocumented feature here, this will require everyone follow this flag, including customers?